### PR TITLE
Feature/eb 1404 add test driver/customer

### PR DESCRIPTION
**Pull request by vdanylov**

### CODE REVIEW

File: src/data/customer/__test__/driver.test.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ import {
   deleteDriverScheduleAsync,
   createDriverScheduleAsync,
   updateDriverScheduleAsync,
+  getDriversWithFiltersAsync,
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: @vdanylov please have a look at this PR from @vinay3206: https://github.com/CarPalFleet/JavaScript-SDK/pull/120
You might want to consider merging his code in your branch and re-run the test suites.

Comment (vdanylov) [reply src/data/customer/__test__/driver.test.js]: @maartenhemmes yeap, we did it in that way, thanks)

File: src/data/account/Account.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {apiResponseErrorHandler} from '../utility/Util';
  * @param {string} email
```
Comment (maartenhemmes) [inline src/data/account/Account.js]: @vdanylov please resolve merge conflicts

File: yarn-error.log [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Arguments: 
-  /usr/local/bin/node /usr/local/Cellar/yarn/1.5.1_1/libexec/bin/yarn.js test -- src/data/notification/__test__/notification.test.js
```
Comment (maartenhemmes) [inline yarn-error.log]: @vdanylov please resolve merge conflicts

File: yarn-error.log [hunk 4]
```diff
@@ -102,11 +102,7 @@ Trace:
   Error: Command failed.
   Exit code: 1
   Command: sh
-<<<<<<< HEAD
```
Comment (maartenhemmes) [inline yarn-error.log]: @vdanylov please resolve merge conflict

File: src/data/customer/__test__/driver.test.js [hunk 5]
```diff
@@ -10,6 +10,9 @@ import {
...
+  getDriverCountsAsync,
+  getDriverRoutesAsync,
 } from '../Driver';
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: @vdanylov snapshot is failing:

● Test getDriverRoutesAsync › should get getDriverRoutesAsync success response

    expect(value).toMatchSnapshot()
    
    Received value does not match stored snapshot 1.

**Pull request merged**